pkgs/servers/nitter/default.nix

```
@@ -7,13 +7,13 @@
 
 nimPackages.buildNimPackage rec {
   pname = "nitter";
-  version = "unstable-2023-04-21";
+  version = "unstable-2023-05-19";
```

```suggestion
  version = "unstable-2023-05-20";
```
(according to https://github.com/zedeus/nitter/commit/e3b3b38a2d43a83b5fc2239ab41e864ee686fb2f)

---

This now makes more sense, thanks :O

With a browser in UTC time, it is showing "May 20, 2023, 12:10 AM UTC".
This is a small detail, we can merge this in EDT timezone unless there is a explicit policy in NixPkgs to use UTC timezone?